Vagrantfile

```diff
@@ -0,0 +1,14 @@
+# -*- mode: ruby -*-
+# vi: set ft=ruby :
+
+Vagrant.configure(2) do |config|
+	config.vm.box = "centos/7"
+	config.vm.synced_folder ".", "/vagrant", disabled: true
+	config.vm.synced_folder "./samples", "/converge_samples"
+	config.vm.provision :converge do |cvg|
```

Comment:
if we're going to use vagrant for integration testing, we probably have to figure out a way to get the correct, local build of converge onto the vagrant machine. This might be best handled by a new option on the vagrant plugin.

also, this looks to be out of date with the latest version of the plugin.